### PR TITLE
Write database schema for counter

### PR DESCRIPTION
**Pull request by jmun10**

### CODE REVIEW

File: app/db.py [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
 
-meta.create_all(engine)
\ No newline at end of file
+
+class Pageloads(Base):
+    __tablename__ = 'pageloads'
+    id = Column(Integer, primary_key=True)
+    timestamp = Column(String(50), nullable=False)
+    page_name = Column(String(50), nullable=False)
+    company = Column(String(50))
```
Comment (michardy) [inline app/db.py]: Please add the attribute `nullable=True`

File: app/db.py [hunk 2]
```diff
@@ -1,8 +1,17 @@
 from sqlalchemy import create_engine, MetaData, Table, Column, Integer, String, DateTime
-# TODO: Add path to table
-engine = create_engine('sqlite://<path_to_DB>', echo = True)
-meta = MetaData()
+from sqlalchemy.ext.declarative import declarative_base
 
-# TODO: Add tables here in PT-5
+# TODO: decide proper place for db
+engine = create_engine('sqlite:///database.db', echo=True)
```
Comment (michardy) [inline app/db.py:5]: The database path will either need to be made relative on both Windows and Linux, docker will need to be used, or a remote database will have to be set up.

**Review by michardy — CHANGES_REQUESTED** [review]
Please fix the database path and add the nullability of company.  It appears nullability might default to true but the ambiguity is a problem.

File: app/db.py (hunk 2 above)
Comment (michardy) [reply app/db.py:5]: So it appears that specifically with SQLAlchemy this is a relative path.  This does not need to be changed.

**Review by michardy — APPROVED** [review]

**Comment by michardy** [comment]
I am realizing we might want to use a [`DateTime`](https://docs.sqlalchemy.org/en/13/core/type_basics.html#sqlalchemy.types.DateTime) object for timestamp.  Particularly if a later goal is to filter by date.

**Review by hyunjoon99 — APPROVED** [review]

**Pull request merged**